processors/gsk/extractors.py

```diff
@@ -19,9 +19,22 @@ def extract_source(record):
 
 
 def extract_trial(record):
+
     # TODO: fix
+    # Get registration date
     registration_date = (
         record['first_received'] or datetime.datetime.now().date())
+
+    # Get gender
+    gender = None
+    if record['gender'] in ['Both', 'Male', 'Female']:
+        gender = record['gender'].lower()
```

Comment:
When a gender can be different than "Both", "Male" or "Female"? It would be good to add a small comment here.